its-incident-app/backend_v2/notify/README.md

```
@@ -0,0 +1,366 @@
+│                               ▼                                 │
+│                       ┌──────────────┐                          │
+│                       │  PostgreSQL  │                          │
+│                       │  (GORM)      │                          │
+│                       └──────────────┘                          │
+│                                                                 │
+└─────────────────────────────────────────────────────────────────┘
+```
+
+## ディレクトリ構成
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)